src/Core/Prefab/TileBuilder.js

```diff
@@ -51,7 +51,9 @@ export default function newTileGeometry(builder, params) {
 
         const geometry = new TileGeometry(params, buffers);
         geometry.OBB = new OBB(geometry.boundingBox.min, geometry.boundingBox.max);
-
+        if (params.hideSkirt) {
+            geometry.setDrawRange(0, geometry.segments * geometry.segments * 2 * 3);//  bufferIndex = (nSeg) * (nSeg) * 2 * 3 (computeBufferTileGeometry.js)
+        }
```

Comment:
This code is fragile. If there is any future change to the geometry computation in `computeBufferTileGeometry#computeBuffers`, this code would then be invalid.

I have two suggestions:
- Create a function in `computeBufferTileGeometry` to compute the draw range of the tile with and without its skirt. This avoid duplication and could be used at vertex creation and when limiting the draw range.
- Since this a code is specific to the tile geometry,  you could maybe move this code in a `TileGeometry#hideSkirt` setter.